packages/ui/src/components/StoryShareLinks/SocialMedia.tsx

```diff
@@ -104,3 +98,16 @@ export function SocialMedia({ className, companyInformation, iconClassName }: Pr
         </div>
     );
 }
+
+export namespace SocialMedia {
+    export type CompanySocials = Pick<
+        NewsroomCompanyInformation,
+        'facebook' | 'instagram' | 'linkedin' | 'tiktok' | 'pinterest' | 'twitter' | 'youtube'
+    >;
```

Comment:
I suggest rewriting these without `Pick`:

```ts
export interface CompanySocials {
   facebook: NewsroomCompanyInformation['facebook'];
   instagram: NewsroomCompanyInformation['instagram'];
   // ...
}
```

You only write it once, but read a thousand times. And with this decomposed format it's easier to read IMO.